config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'cv_user_main_details/create'
+  get 'cv_user_main_details/update'
```

Comment:
@AkshayPashnathSuresh 
> - route naming is not good. change it.
> - change get to resources

Reply:
@AkshayPashnathSuresh 
confirm that requested changes are already fixed before submitting for review again :P 
- route naming is not good. change it. THIS IS NOT DONE YET